src/components/dashboard/subscriptions/Subscription.js

```diff
@@ -1,4 +1,4 @@
-import React, {Component, PropTypes} from 'react';
+import React, {Component} from 'react';
```

Comment:
shouldn't we add prop validation rather than removing the unused proptype import?